fastlane/lib/fastlane/actions/create_keychain.rb

```diff
@@ -48,6 +57,11 @@ def self.available_options
                                        env_name: "KEYCHAIN_NAME",
                                        description: "Keychain name",
                                        optional: false),
+          FastlaneCore::ConfigItem.new(key: :is_path,
```

Comment:
Rather than setting -is_path and passing in a keychain via -name, why not create a -path option and pass the keychain path in that way? Seems simpler than having to set a boolean value on the command and then use a inaccurately name parameter ("name"). Thoughts?